docs/retrieve.txt

```diff
@@ -0,0 +1,360 @@
+to only return documents that meet these requirements. To run the query, call the ``where()``
+method on an Eloquent model or query builder that represents your collection.
+
+You can pass the following types of query filters to the ``where()`` method:
+
+- ``where('<field name>', <value>)``: instructs MongoDB to return documents that match
+  the exact value for the specified field name
+
+- ``where('<field name>', '<comparison operator>', <value>)``: instructs MongoDB to
+  return documents in which the field values meet the comparison criteria
+
+To apply multiple sets of criteria to the find operation, you can chain a series
+of ``where()`` methods together.
+
+.. _laravel-retrieve-eloquent:
+
+Use Eloquent Models to Retrieve Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use Laravel's Eloquent object-relational mapper (ORM) to create models that represent
+MongoDB collections. To retrieve documents from a collection, call the ``where()`` method
+on the collection's corresponding Eloquent model.
+
+This example calls two ``where()`` methods on the ``Movie`` Eloquent model to retrieve
+documents that meet the following criteria:
+
+- ``year`` field has a value of ``2010``
+- ``imdb.rating`` nested field has a value greater than ``8.5``
+
+.. important:: Required File Changes
+
+   To run the code example, you must edit the ``MovieController.php`` file in your
+   Laravel Quick Start application. Copy the code example and paste it into the ``show()``
+   function, replacing any existing code inside this function.
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: php
+
+      return view('browse_movies', [
+         'movies' => Movie::where('year', 2010)
+            ->where('imdb.rating', '>', 8.5)
+            ->get()
+      ]);
+
+   .. output::
+      :language: none
+      :visible: false
+
+      Title: Inception
+      Year: 2010
+      Runtime: 148
+      IMDB Rating: 8.8
+      IMDB Votes: 1294646
+      Plot: A thief who steals corporate secrets through use of dream-sharing
+      technology is given the inverse task of planting an idea into the mind of a CEO.
+
+      Title: Senna
+      Year: 2010
+      Runtime: 106
+      IMDB Rating: 8.6
+      IMDB Votes: 41904
+      Plot: A documentary on Brazilian Formula One racing driver Ayrton Senna, who won the
+      F1 world championship three times before his death at age 34.
+
+.. _laravel-retrieve-query-builder:
+
+Use Laravel Queries to Retrieve Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use Laravel's database query builder to run find operations instead of using Eloquent
+models. To run the database query, import the ``DB`` facade into your controller file and use
+Laravel's query builder syntax.
+
+The following example uses Laravel's query builder to retrieve documents in which the value
+of the ``imdb.votes`` nested field is greater than ``1300000``.
```

Comment:
The query must be directly after this introducing sentence. And move the "important" block after.

And explain why this changes are necessary: because the data is returned as an array instead of instances of the Model class.